docker/start.sh

```diff
@@ -23,6 +23,9 @@ case "$BOT_BINARY" in
     cleaner*)
         ./cleaner
         ;;
+    all*)
+        ./el_monitorro & ./sync & ./deliver & ./cleaner
```

Comment:
I'm running the bot on my private vps and sometimes binaries just stop because of failures. For example:
```
Apr 14 02:27:17 vps695005 bash[1090]: [2021-04-14T00:27:17Z ERROR telegram_bot::api] error=Bad Gateway
Apr 14 02:27:17 vps695005 bash[1090]: [2021-04-14T00:27:17Z ERROR telegram_bot::stream] request error error=Bad Gateway
Apr 14 02:27:17 vps695005 bash[1090]: [2021-04-14T00:27:17Z ERROR el_monitorro] Couldn't start a bot
Apr 14 02:27:18 vps695005 systemd[1]: el_monitorro.service: Service hold-off time over, scheduling restart.
Apr 14 02:27:18 vps695005 systemd[1]: el_monitorro.service: Scheduled restart job, restart counter is at 4.
Apr 14 02:27:18 vps695005 systemd[1]: Stopped El Monitorro bot.
Apr 14 02:27:18 vps695005 systemd[1]: Started El Monitorro bot.
```
I set systemd service so the service is restarted on every failure. Is it possible to restart stopped binaries inside of  a docker container?

Reply:
There really isn't a great way. I'm not sure if one executable crashing would kill the entire container or not. If so, it could be resolved with having docker auto-restart. Another solution would be to have the docker restart every `x` hours or so to keep all the executables running properly.

Reply:
@m0ngr31 Can you leave a comment about it and your pr is good to merge. Will you need a new release on docker hub?

Reply:
Yeah that'd be great :)

I'll get the README updated